<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_LienJeux</class>
 <widget class="QDialog" name="D_LienJeux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lien entre 2 jeux</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QPushButton" name="Bt_OK">
     <property name="text">
      <string>O&amp;K</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="Bt_Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="HLay_Jeux"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Saisir le nom ou le code du jeux :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ce jeu est :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Cbx_DescriptifJeu">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Choisir le type de lien à établir entre les 2 jeux :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="Cbx_TypeLien"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
